gslib/utils/text_util.py

```diff
@@ -301,22 +301,37 @@ def ttyprint(*objects, **kwargs):
     data += end
     ttywrite(file, data)
   else:  # PY3
-    pref_enc = locale.getpreferredencoding(False)
-    if isinstance(sep, str):
-      sep = sep.encode(pref_enc)
-    if isinstance(end, str):
-      end = end.encode(pref_enc)
-    byte_objects = []
-    for object in objects:
-      if isinstance(object, bytes):
-        byte_objects.append(object)
-      elif isinstance(object, str):
-        byte_objects.append(object.encode(pref_enc))
-      else:
-        byte_objects.append(str(object).encode(pref_enc))
-    data = sep.join(byte_objects)
-    data += end
-    ttywrite(file, data)
+    def Py3Print(encoding, sep=sep, end=end):
+      """ Passing in encoding allows for a try-except block to attempt the
+      default encoding and then fall back to utf-8 on failure to encode
+
+      Args:
+        encoding: Encoding to be used for all encode operations while printing.
+        sep: Line separator to be used for printing (defined in outer scope).
+        end: End character to be used for printing (defined in outer scope).
+      Return: None
```

Comment:
Need an extra line of whitespace between end of `Args` and start of `Return`.  Also, `Return` should be `Returns`, or I think the linter will whine.

Reply:
Whitespace adjusted and changed `Return` to `Returns`